### PR TITLE
Added zap for Aluxian-Messenger

### PR DESCRIPTION
**Pull request by adidalal**
As comment, adds a zap stanza for Messenger.

### CODE REVIEW

File: Casks/aluxian-messenger.rb [hunk 1]
```diff
@@ -9,4 +9,10 @@
   license :oss
 
   app 'Messenger.app'
+  
+  zap :delete => [
+                  '~/Library/Preferences/com.aluxian.messengerfordesktop.plist'
```
Comment (vitorgalvao) [inline Casks/aluxian-messenger.rb]: Missing a `,`.

Comment (adityadalal924) [inline Casks/aluxian-messenger.rb]: Guess I should really run things through a linter, whoops. Sorry about that

Comment (vitorgalvao) [inline Casks/aluxian-messenger.rb]: Running the audit would catch these errors: `brew cask audit {{cask_name}}`.

**Comment by vitorgalvao** [comment]
Why close this?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 6f2cef9a9eb0b16f623ded247ea371f31c3f2214. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by adityadalal924** [comment]
Going to actually clone the repo and make proper branches instead of just using the web interface from now on.

Was going to resubmit with the fix, but I see it's already done. Thanks!

**Pull request closed without merging**